YAPA/MainWindow.xaml.cs

```diff
@@ -598,7 +586,9 @@ private void Stop_Click(object sender, RoutedEventArgs e)
             else
             {
                 ResetTicking();
+                _isWork = false;
                 PlayMusic();
+                _isWork = true;
```

Comment:
This looks a bit strange. Maybe it would be better to have 3 functions: PlayBreakMusic(), PlayWorkMusic() and  PlayMusic() that would invoke one of these functions.